index.html

```diff
@@ -0,0 +1,258 @@
+                <p class="card-primarytext">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="tags d-flex">
+                    <li>HTML</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul> 
+                <button class="button-see-project" type="button">See Project</button>
+            </h2>
+        </h2>
+    </h2>
+</section>
+<section class="about app-bckground-color d-flex">
+    <h2 class="about-myself app-margin">
+        <h2 class="about-primary-text-0"
+        >About myself 
+        </h2>      
+        <p class="about-primary-text-2">
+            Hello I’m a software developer! I can help you build a product, feature, or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+        </p>
+        <h3 class="headline-letsconnect">
+            LET’S CONNECT
+        </h3>
+        <ul class="about-myself-social-media-icons">
+            <li><a href="#"><img src="twitter-icon.png" alt="Twitter"></a></li>
+            <li><a href="#"><img src="Linkedin-icon.png" alt="LinkedIn"></a></li>
+            <li><a href="#"><img src="mail-icon.png" alt="Email"></a></li>
+            <li><a href="#"><img src="git-icon.png" alt="GitHub"></a></li>
+            <li><a href="#"><img src="chill-icon.png" alt="Chill"></a></li>
+        </ul>
+        <div><button class="button-see-project-2" type="button">Get My Resume</button></div>
+        <div class="about-body app-margin">
+            <h2>Languages</h2>
+            <ul class="technologies">
+                <li>
+                    <img src="Ellipse 1.png" alt="Icon">
+                    <span>JavaScript</span>
+                </li>
+                <li>
+                    <img src="Ellipse 2.png" alt="Icon">
+                    <span>HTML</span>
+                </li>
+                <li>
+                    <img src="Ellipse 3.png" alt="Icon">
+                    <span>CSS</span>
+                </li>
+            </ul>
+        </div>
+  <li class="resume-list frameworks">
+    <div class="lang-container">
+      <details>
+        <summary>
+          Frameworks
+        </summary>
+        <ul class="lang-list">
+          <li class="skill-item">
+            N/A
+          </li>
+        </ul>
+      </details>
+    </div>
+  </li>
+  <li class="resume-list skills">
+    <div class="lang-container">
+      <details>
+        <summary>
+           Skills
+        </summary>
+        <ul class="lang-list">
+          <li class="skill-item">
+            HTML, CSS, and Javascript
+          </li>
+          <li class="skill-item">
+             Git and Github
+          </li>
+          <li class="skill-item">
+            Python
+          </li>                   
+        </ul>                   
+    </div>
+  </li>
+</ul>
```

Comment:
- [ ] As requested by the previous reviewer you should stick to the design as much as possible like color, font, size etc. Please check the design below and please implement it.
### Your design
![his](https://github.com/wachirasenior0/portfolio-setup-and-mobile-first/assets/81727988/8fabc839-68d6-4f05-9061-fb5b7a526d8e)

### Figma design
![figma](https://github.com/wachirasenior0/portfolio-setup-and-mobile-first/assets/81727988/aeddc5ad-83e3-4c64-b24c-0adf72020303)